package.json

```diff
@@ -37,29 +37,38 @@
 	"dependencies": {
 		"ansicolors": "0.3.2",
 		"app-module-path": "2.2.0",
-		"axios": "0.16.2",
+		"axios": "^0.16.2",
```

Comment:
Thank you for response, I did not know it before removed axios from project and got build errors